file_checker.py

```diff
@@ -178,7 +178,7 @@ def extract_info(fpath):
 
 
 def checkFile(file):
-    model = joblib.load("model/model.pkl")
+    model = joblib.load("model/model_new.pkl")
```

Comment:
Comment this line and add a comment to uncomment this line to use the new model in the program.